### PR TITLE
[BULK] DocuTune - Rebranding of Azure Cosmos DB APIs

### PR DESCRIPTION
**Pull request by alexbuckgit**
DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review and merge within five business days or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 1c5ab0c9-80d8-4820-b550-6b2ddb22d568
InitiativeId: docutune-cosmos-db-rebrand-2022-10
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by ajcvickers** [comment]
Some of these changes don't look correct and/or are not consistent. Please don't merge until we can verify that the correct terminology is being used.

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by seesharprun** [comment]
Happy to review. I saw a few that went from "SQL API" to "API for NoSQL" which is correct. Also "Cosmos DB" must always be prefixed with "Azure".

**Comment by alexbuckgit** [comment]
Hi @ajcvickers - I think this PR is ready to merge. Thanks!

#docutune-request-approval

File: entity-framework/core/what-is-new/ef-core-5.0/breaking-changes.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ The following API and behavior changes have the potential to break existing appl
...
-| [Cosmos: `id` property renamed to `__id`](#cosmos-id)                                                                                 | Low        |
-| [Cosmos: byte[] is now stored as a base64 string instead of a number array](#cosmos-byte)                                             | Low        |
-| [Cosmos: GetPropertyName and SetPropertyName were renamed](#cosmos-metadata)                                                          | Low        |
+| [Azure Cosmos DB: Partition key is now added to the primary key](#cosmos-partition-key)                                                        | Low        |
+| [Azure Cosmos DB: `id` property renamed to `__id`](#cosmos-id)                                                                                 | Low        |
+| [Azure Cosmos DB: byte[] is now stored as a base64 string instead of a number array](#cosmos-byte)                                             | Low        |
+| [Azure Cosmos DB: GetPropertyName and SetPropertyName were renamed](#cosmos-metadata)                                                          | Low        |
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-5.0/breaking-changes.md:29]: These need to be reformatted to account for the extra length

File: entity-framework/core/what-is-new/ef-core-6.0/breaking-changes.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ EF Core 6.0 targets .NET 6. Applications targeting older .NET, .NET Core, and .N
 | [Added IReadOnly Metadata interfaces and removed extension methods](#ireadonly-metadata)                                            | Low        |
 | [IExecutionStrategy is now a singleton service](#iexecutionstrategy)                                                                | Low        |
 | [SQL Server: More errors are considered transient](#transient-errors)                                                               | Low        |
-| [Cosmos: More characters are escaped in 'id' values](#cosmos-id)                                                                    | Low        |
+| [Azure Cosmos DB: More characters are escaped in 'id' values](#cosmos-id)                                                                    | Low        |
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-6.0/breaking-changes.md:35]: Also here

**Review by ajcvickers — APPROVED** [review]

**Pull request closed without merging**